daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/transaction/AuthorizationChecker.scala

```diff
@@ -9,7 +9,39 @@ import com.daml.lf.ledger.Authorize
 import com.daml.lf.ledger.FailedAuthorization
 import com.daml.lf.transaction.Node
 
-private[lf] object CheckAuthorization {
+trait AuthorizationChecker {
```

Comment:
```suggestion
abstract class AuthorizationChecker {
```

Reply:
Sure, can do that. Why are you asking for this?